### PR TITLE
Task J-217: timestamps not show without record in comment list

### PR DESCRIPTION
**Pull request by vitalik-prog**

### CODE REVIEW

File: frontend/src/components/common/comments-list/comments-list.tsx [hunk 1]
```diff
@@ -3,15 +3,17 @@ import { Comment } from 'common/types/types';
 import styles from './styles.module.scss';
 
 type Props = {
+  hasRecord?: boolean;
```
Comment (Denidzio) [inline frontend/src/components/common/comments-list/comments-list.tsx]:
```suggestion
  hasRecord: boolean;
```
Why do you use 'undefined' type here?

**Review by Denidzio — DISMISSED** [review]

File: frontend/src/components/common/comments-list/comments-list.tsx (hunk 1 above)
Comment (what1s1ove) [reply frontend/src/components/common/comments-list/comments-list.tsx]: it's okay because we can have cases when we don't have an record

Comment (what1s1ove) [inline frontend/src/components/common/comments-list/comments-list.tsx]:
```suggestion
  hasTimestemps?: boolean;
```

File: frontend/src/components/common/comment-item/comment-item.tsx [hunk 2]
```diff
@@ -7,11 +7,12 @@ import { Link } from 'components/common/common';
 import { AppRoute } from 'common/enums/enums';
 
 type Props = {
+  hasRecord?: boolean;
```
Comment (what1s1ove) [inline frontend/src/components/common/comment-item/comment-item.tsx]:
```suggestion
  hasTimestemp?: boolean;
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/comments-list/comments-list.tsx (hunk 1 above)
Comment (vitalik-prog) [reply frontend/src/components/common/comments-list/comments-list.tsx]: Yes, in live record, we have comments but don't have record

File: frontend/src/components/common/comment-item/comment-item.tsx [hunk 3]
```diff
@@ -7,11 +7,12 @@ import { Link } from 'components/common/common';
 import { AppRoute } from 'common/enums/enums';
 
 type Props = {
+  hasTimestamps?: boolean;
```
Comment (what1s1ove) [inline frontend/src/components/common/comment-item/comment-item.tsx]:
```suggestion
  hasTimestamp?: boolean;
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**